setup.py

```diff
@@ -50,6 +50,7 @@ def check_setuptools_features():
     'pytest>=3.0.0',
     'pytest-catchlog>=1.2.2',
     'pytest-cov>=2.2.1',
+    'pytest-mock',
```

Comment:
Is this an extension of `unittest.mock`? Just wondering if we can stick with the one from the standard library.